The exr crate doesn't support big-endian and doesn't compile there.
Disable it to avoid compile-time breakage on sparc64.

Index: modcargo-crates/image-0.24.1/Cargo.toml
--- modcargo-crates/image-0.24.1/Cargo.toml.orig
+++ modcargo-crates/image-0.24.1/Cargo.toml
@@ -128,7 +128,7 @@ avif-encoder = ["ravif", "rgb"]
 benchmarks = []
 bmp = []
 dds = ["dxt"]
-default = ["gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "dds", "farbfeld", "jpeg_rayon", "openexr"]
+default = ["gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "dds", "farbfeld", "jpeg_rayon"]
 dxt = []
 farbfeld = []
 hdr = ["scoped_threadpool"]
